app/models/supplejack_api/concerns/user_set.rb

```diff
@@ -95,6 +95,28 @@ def self.featured_sets(num = 16)
     def update_attributes_and_embedded(new_attributes = {}, user = nil)
       new_attributes = new_attributes.try(:symbolize_keys) || {}
 
+      update_set_items(new_attributes)
+      update_featured_set(new_attributes, user)
+
+      self.attributes = new_attributes
+
+      self.description = '' if description.nil?
+      self.approved = false if approved.nil?
+
+      save
```

Comment:
ok will change this, although the original code had just save.   I'll see if it still works